openshift/system/Dockerfile

```diff
@@ -27,11 +23,16 @@ RUN yum-config-manager --save --setopt=cbs.centos.org_repos_sclo7-rh-ruby26-rh-c
               postgresql10 postgresql10-devel postgresql10-libs \
               file \
               rh-nodejs12 \
-    && rpm -i /tmp/sphinxsearch.rpm \
-    && rm /tmp/*.rpm \
     && yum install -y epel-release \
     && yum -y clean all
 
+# Install sphinx search
+ARG SPHINX_VERSION=3.4.1
+COPY openshift/system/sphinx-${SPHINX_VERSION}-efbcc65-linux-amd64.tar.gz /tmp/sphinxsearch.tar.gz
+RUN tar zxf /tmp/sphinxsearch.tar.gz -C /tmp \
+    && cp /tmp/sphinx-${SPHINX_VERSION}/bin/* /usr/bin/ \
```

Comment:
Is `tar + cp` a manual installation similar to what `rpm` does? Any particular reason why we're not using an rpm anymore?

Reply:
I don't see RPM download options available anymore.

Reply:
> Is `tar + cp` a manual installation

It is.

> reason why not using an rpm

Because there's no official RPM available for Sphinx Search [v3](sphinxsearch.com/downloads/current) as we used to have for [v2](http://sphinxsearch.com/downloads/archive).